pkgs/development/python-modules/smhi/default.nix

```
@@ -0,0 +1,55 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, aiohttp
+, black
+, coverage
+, flake8
+, mock
+, pyflakes
+, pylint
+, pytest
+, pytest-asyncio
+, pytest-cov
+, requests
+, requests-toolbelt
+}:
+
+buildPythonPackage rec {
+  pname = "smhi";
+  version = "1.0.13";
+
+  src = fetchPypi {
+    pname = "smhi_pkg";
+    inherit version;
+    sha256 = "1mp13vnr7c8ll0yvxdn1dsd10l1j7zdcgkk777w6lmhc7as7lqvj";
+  };
+
+  propagatedBuildInputs = [
+    aiohttp
+    requests
+    requests-toolbelt
```

It looks like that the project moved from `requests` to `aiohttp` at some point.

```suggestion
    aiohttp
```